Products/PasswordStrength/__init__.py

```diff
@@ -1,26 +1,26 @@
-
-"""PasswordStrength
-"""
-
-__author__ = "Dylan Jay <software@pretaweb.com"
-
-from AccessControl.Permissions import add_user_folders
-from Products.PluggableAuthService import registerMultiPlugin
-from plugin import PasswordStrength, \
-                   manage_addPasswordStrength, \
-                   manage_addPasswordStrengthForm
-
-def initialize(context):
-    """Initialize the PasswordStrength plugin."""
-    registerMultiPlugin(PasswordStrength.meta_type)
-    
-    context.registerClass(PasswordStrength,
-                          permission=add_user_folders,
-                          constructors=(manage_addPasswordStrengthForm,
-                                        manage_addPasswordStrength),
-                          #icon='www/noduplicatelogin.png',
-                          visibility=None,
-                          )
-
-
-PROJECTNAME = "PasswordStrength"
+
+"""PasswordStrength
+"""
+
+__author__ = "Dylan Jay <software@pretaweb.com"
```

Comment:
It seems odd to specify this in every file. I know you'e just maintaining the pattern that this library uses, but perhaps the original author will be okay with you removing it as long as it stays here in __init__.

I'd also expect to see __version__ here as well.